scripts/rmc.py

```diff
@@ -17,7 +17,7 @@ def is_exe(name):
 
 
 def dependencies_in_path():
-    for program in [RMC_RUSTC_EXE, "symtab2gb", "cbmc", "goto-instrument"]:
+    for program in [RMC_RUSTC_EXE, "symtab2gb", "cbmc", "cbmc-viewer", "goto-instrument", "goto-cc"]:
```

Comment:
We need to update CI to install this program

Reply:
Todo: Add to readme!